fluss-connectors/fluss-connector-flink/src/main/java/com/alibaba/fluss/connector/flink/source/reader/FlinkSourceSplitReader.java

```diff
@@ -262,17 +287,42 @@ public Set<TableBucket> removePartitions(Map<Long, String> removedPartitions) {
         // todo, may consider to close the current snapshot reader if
```

Comment:
The todo comment can be removed now since in this pr, we wil close currentBoundedSplit